core/src/main/java/com/amplifyframework/datastore/DataStoreChannelEventName.java

```diff
@@ -52,7 +52,22 @@
     /**
      * The remote synchonization processes stopped.
      */
-    REMOTE_SYNC_STOPPED("remote_sync_stopped");
+    REMOTE_SYNC_STOPPED("remote_sync_stopped"),
```

Comment:
Should we also remove `REMOTE_SYNC_STARTED` and `REMOTE_SYNC_STOPPED` as part of this? Right now, I think we have some duplicative code. A good goal would be to align the events to iOS and Android. In so doing, that means we have to rename/remove the events we have, that already provide a similar functionality.

Reply:
Makes sense. We're using this for a slightly different purpose I believe (it's more of a orchestrator start and stop), but I don't believe anyone is using this as it is not documented.

Reply:
I removed REMOTE_SYNC_STOPPED and REMOTE_SYNC_STARTED